course-content/30_bcdata.qmd

```diff
@@ -0,0 +1,233 @@
+)
+hospital_filtered.head()
+```
+
+:::{.callout-tip}
+## SQL Syntax
+
+In the query above, note that the column in the query is NOT put in an extra set of quotations, whereas the column value is for character columns. This is in line with standard SQL syntax, but if you are not familiar with SQL then it might look a little confusing at first! General rule of thumb for the query is that it will be something of the form:
+
+`"COLUMN_A = 'some string value' OR COLUMN_B = 42"`
+:::
+
+## Geospatial Plots
+
+Alright, we have some geospatial data in Python now. That's great. I really love 
+
+POLYGON ((-118.37713213 49.91303648, -118.37772348 49.91308665, -118.37819007 49.91293438, -118.37864102 49.91273297, -118.37885065 49.91240661, -118.37894791 49.91201733, -118.3791819 49.91159333, -118.37960692 49.91128489, -118.3800468 49.91108959, -118.38056154 49.91089751, -118.38113866 49.91078258, -118.38179033 49.91067428, -118.38237742 49.91061745, -118.38289869 49.91056691, -118.38332771 49.91048198, -118.38381897 49.91036744, -118.38447835 49.91018407, -118.38517458 49.91007327, -118.38585128 49.91001193, -118.38650275 49.90998635, -118.38735549 49.90993369, -118.38790802 49.90991568, -118.38842369 49.90990548, -118.38892747 49.90988955, -118.38972597 49.90985294, -118.39035971 49.90986504, -118.39098773 49.9099297, -118.39161337 49.90996627, -118.39214558 49.91004415, -118.39273105 49.91006478, -118.3932995 49.90999734, -118.39412967 49.90993219, -118.39464161 49.90987966, -118.39535483 49.90980076, -118.39601118 49.90972691, ...) 
+
+this time of year, don't you? 
+
+... no? No clue where that is or what it might look like? Me neither. Luckily for us, we can checkout these datasets with some geospatial plots! The geopandas dataframe that the data was loaded into can automatically display a map with points, borders, or coloured regions. 
+
+Let's look at the basic plot that is displayed for a 'point' style geometry vs. a 'polygon' style geometry. Don't forget to import matplotlib to display the plots! 
+
+```{python}
+#| layout-ncol: 2
+import matplotlib.pyplot as plt
+
+hospitals.plot()
+plt.show()
+
+fires.plot()
+plt.show()
+```
+
+Way better! With polygon areas, we also have a few more options for how we wish to colour the regions. 
+
+We can colour based on a categorical variable:
+
+```{python}
+fires.plot(column='MOF_FIRE_CENTRE_NAME')
+plt.show()
+```
+
+Or on a numerical variable:
+
+```{python}
+fires.plot(column='FEATURE_AREA_SQM', legend=True)
+plt.show()
+```
+
+Or if we only want to display the boundaries of our regions, we can do that too:
+
+```{python}
+fires.boundary.plot()
+plt.show()
+```
+
+You'll notice that we can easily see the shape of B.C. in these region plots, but it is less obvious for the hospitals dataset. Beceause the hospitals dataset only contains 'point' objects, it will just place these points on an axis, but with no region boundaries to reference, it might just look like a scattering of points on a plot. 
```

Comment:
```suggestion
You'll notice that we can easily see the shape of B.C. in these region plots, but it is less obvious for the hospitals dataset. Because the hospitals dataset only contains 'point' objects, it will just place these points on an axis, but with no region boundaries to reference, it might just look like a scattering of points on a plot. 
```